File: app/src/main/java/com/neome/java/api/core/base/msg/MsgEntUserIdSet.java
// neome.ai API - do not change
//

package com.neome.java.api.core.base.msg;

import com.neome.java.api.meta.base.Types.EntUserId;
import com.neome.java.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgEntUserIdSet extends Msg
{
  @Nullable
  public EntUserId entUserId;

  @Nullable
  public EntUserId[] entUserIdSet;
}

File: app/src/main/java/com/neome/api/core/base/msg/MsgEntUserIdSet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.base.msg

import com.neome.api.meta.base.Types.EntUserId
import com.neome.api.nucleus.base.msg.Msg

class MsgEntUserIdSet : Msg() {
    var entUserId: EntUserId? = null
    var entUserIdSet: EntUserId[]? = null
}
